pkg/sink/codec/canal/java.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package internal
+package canal
```

Comment:
Moving to sink/common is better because Debezium also uses it.

Reply:
why the debezium need this file ?

Reply:
`jdbcType` field for DDL event